openedx/core/djangoapps/content_staging/filters.py

```diff
@@ -49,7 +49,7 @@ def remove_large_data(fd: StagedContentFileData):
             """ Remove 'data' from the immutable StagedContentFileData object, if it's above the size limit """
             if fd.data and len(fd.data) > limit:
                 # these data objects are immutable so make a copy with data=None:
-                return StagedContentFileData(**{**asdict(fd), "data": None})
+                return StagedContentFileData(**{**asdict(fd), "data": None})  # type: ignore
```

Comment:
Not sure why it didn't like this line, but I'm planning to remove this filter implementation soon anyways. So just ignored it for now.